yb-voyager/cmd/common_test.go

```diff
@@ -0,0 +1,396 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/migassessment"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
+)
+
+func TestAssessmentReportStructs(t *testing.T) {
+	excpectedDBObject := struct {
+		ObjectType   string `json:"ObjectType"`
+		TotalCount   int    `json:"TotalCount"`
+		InvalidCount int    `json:"InvalidCount"`
+		ObjectNames  string `json:"ObjectNames"`
+		Details      string `json:"Details,omitempty"`
+	}{}
+
+	t.Run("Check DBObject structure", func(t *testing.T) {
+		utils.CompareStructs(t, reflect.TypeOf(utils.DBObject{}), reflect.TypeOf(excpectedDBObject), "DBObject")
+	})
+
+	// Define the expected structure for utils.SchemaSummary
+	expectedSchemaSummary := struct {
+		Description string           `json:"Description"`
+		DBName      string           `json:"DbName"`
+		SchemaNames []string         `json:"SchemaNames"`
+		DBVersion   string           `json:"DbVersion"`
+		Notes       []string         `json:"Notes,omitempty"`
+		DBObjects   []utils.DBObject `json:"DatabaseObjects"`
+	}{}
+
+	t.Run("Check SchemaSummary structure", func(t *testing.T) {
```

Comment:
Any particular reason for calling `t.Run`. Usually you use `t.Run` when you have to write tests dynamically, or run them in parallel. 

Here, ideally, a separate test case function for each structure should be fine, right?

Reply:
Well mostly this was grouping the tests(for example all the tests for smaller structs related to a particular struct) under one function. 
If you feel that the individual function approach is better then that can be done I guess.